pkg/generated/openapi/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Does federation go in here?  Because we've tried to treat federation as distinct, and combining the openapi docs here but everything else be separate seems wrong. 

Package directory is fine, although @thockin almost certainly has opinions.